tools/create_release_files.sh

```diff
@@ -36,57 +58,59 @@ if [[ `basename $CURR_DIR` != "tools" ]] ; then
   exit 1
 fi
 
+##
+## Variables with defaults (if not overwritten by environment)
+##
 GPG_PASSPHRASE=${GPG_PASSPHRASE:-XXX}
 GPG_KEY=${GPG_KEY:-XXX}
+GIT_AUTHOR=${GIT_AUTHOR:-"Your name <you@apache.org>"}
 GIT_BRANCH=${GIT_BRANCH:-branch-1.0}
 OLD_VERSION=${OLD_VERSION:-0.6-incubating-SNAPSHOT}
 RELEASE_VERSION=${NEW_VERSION}
 RELEASE_CANDIDATE=${RELEASE_CANDIDATE:-rc1}
 NEW_VERSION_HADOOP1=${NEW_VERSION_HADOOP1:-"$RELEASE_VERSION-hadoop1"}
-USER_NAME=${USER_NAME:-pwendell}
+USER_NAME=${USER_NAME:-yourapacheidhere}
 MVN=${MVN:-mvn}
 GPG=${GPG:-gpg}
-SHASUM=${SHASUM:-sha512sum}
-MD5SUM=${MD5SUM:-md5sum}
-sonatype_user=${sonatype_user:-rmetzger} #legacy variable name referring to maven
+sonatype_user=${sonatype_user:-yourapacheidhere}
 sonatype_pw=${sonatype_pw:-XXX}
 
-#echo $NEW_VERSION_HADOOP1
-#sleep 5
-#set -e
 
-# create source package
+if [ "$(uname)" == "Darwin" ]; then
+    SHASUM=shasum
+    MD5SUM=md5
```

Comment:
Where does it state, this is the desired output? For consistency, I probably that makes sense.